packages/server/test/_testHelpers.ts

```diff
@@ -53,7 +53,7 @@ export function routerToServerAndClient<TRouter extends AnyRouter>(
     wss,
     router,
     createContext: ({ req, res }) => ({ req, res }),
-    ...(opts?.wssServer ?? {}),
+    ...(opts?.wssServer as any ?? {}),
```

Comment:
I had to type this as `any` because TS complains about the optional `createContext` prop coming from `Partial<WSSHandlerOptions<TRouter>>` (at least I think that's what it is) although the error doesn't happen with `Partial<CreateHttpHandlerOptions<TRouter>>` for some reason :thinking:

Reply:
that's fine, i'm not sacred of the test code